### PR TITLE
Sharp minima can generalize - Marco

### PR DESCRIPTION
**Pull request by MarcoArmenta**

### CODE REVIEW

File: article/_posts/2020-06-05-SharpMinima.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+![](/article/images/SharpMinima/thm5a.jpg)    
+
+![](/article/images/SharpMinima/thm5b.jpg)    
+
+What they prove now is that the notion
+
+![](/article/images/SharpMinima/max.jpg)    
+
+for sharpness can also be modified with the maps $$T_\alpha$$.
```
Comment (AntoineTheb) [inline article/_posts/2020-06-05-SharpMinima.md:56]: "map" or "maps" ? Is there more than one mapping $$T_\alpha$$ ?

Comment (MarcoArmenta) [reply article/_posts/2020-06-05-SharpMinima.md:56]: maps, there is a map for every alpha.

File: article/_posts/2020-06-05-SharpMinima.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+* There is also a notion of flatness concerning the Hessian, namely the spectral norm and trace of the Hessian. These two have information from the eigenvectors of the Hessian.
+
+## Properties of ReLU
+
+![](/article/images/SharpMinima/relu.jpg)    
+
+![](/article/images/SharpMinima/def5.jpg)    
+
+* **The map $$T_\alpha$$ produces a network with the same network function**.
```
Comment (AntoineTheb) [inline article/_posts/2020-06-05-SharpMinima.md:36]: "maps"

Comment (MarcoArmenta) [reply article/_posts/2020-06-05-SharpMinima.md]: Done

**Comment by AntoineTheb** [comment]
I think you should add an introduction and expand on the conclusion. I had to go to the paper to see why flatness vs. sharpness might be important and why the title mentioned generalisation but the review didn't.

**Comment by AntoineTheb** [comment]
Also, I noticed that you keep force-pushing your changes instead of just adding them to your branch. This goes against git's purpose of keeping an history of changes; see https://www.atlassian.com/git/tutorials/syncing/git-push for instructions on pushing

**Comment by MarcoArmenta** [comment]
@AntoineTheb  I made a couple of changes. What do you think?

**Pull request merged**